### PR TITLE
Handle reruns of the plugin

### PR DESCRIPTION
**Pull request by ludeeus**
This will make sure that if the label `needs-rebase` exist we don't add another message about it.

ref: <https://github.com/home-assistant/home-assistant.io/pull/12852>

### CODE REVIEW

File: src/plugins/DocsTargetBranch/docs_target_branch.ts [hunk 1]
```diff
@@ -88,13 +96,16 @@ const wrongTargetBranchDetected = async (
...
 
-  promises.push(
-    context.github.issues.createComment(
-      context.issue({
-        body,
-      })
-    )
-  );
+  if (!currentLabels.includes("needs-rebase")) {
+    // If the label "needs-rebase" allready exsist we can assume that this action has run, and we should ignore it.
```
Comment (balloob) [inline src/plugins/DocsTargetBranch/docs_target_branch.ts]:
```suggestion
    // If the label "needs-rebase" already exsist we can assume that this action has run, and we should ignore it.
```

Comment (balloob) [reply src/plugins/DocsTargetBranch/docs_target_branch.ts]: Shouldn't we just cancel the whole operation if it has this tag, assuming that all other work has also already been done?

Comment (ludeeus) [reply src/plugins/DocsTargetBranch/docs_target_branch.ts]: 🤦

File: src/plugins/DocsTargetBranch/docs_target_branch.ts [hunk 2]
```diff
@@ -1,100 +1,112 @@
...
-    context.github.issues.createComment(
-      context.issue({
-        body,
-      })
-    )
-  );
-
-  await Promise.all(promises);
-};
+import { Application } from "probot";
```
Comment (balloob) [inline src/plugins/DocsTargetBranch/docs_target_branch.ts]: What happened that every line has changed ?

**Review by balloob — APPROVED** [review]

File: src/plugins/DocsTargetBranch/docs_target_branch.ts (hunk 2 above)
Comment (ludeeus) [reply src/plugins/DocsTargetBranch/docs_target_branch.ts]: Probably line endings

Comment (ludeeus) [reply src/plugins/DocsTargetBranch/docs_target_branch.ts]: Yup, that was it, had changed to CRLF, moved it back now and it looks clean again.

**Pull request merged**